packages/mdc-bottom-line/_mixins.scss

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Public
+
+@mixin mdc-bottom-line-color($color) {
+  .mdc-bottom-line {
```

Comment:
now that this is a public component, we should not include `.mdc-bottom-line` in the public mixins. I think this is unnecessarily specific. We should assume the client calling these mixins has already selected the relevant `mdc-bottom-line` elements with a CSS selector.

Reply:
Agreed, but users won't be able to color the bottom-line from the parent like we currently do. I think we should leave it as is, or add a mixin from the text-field mixins
```
@mixin mdc-text-field-focused-bottom-line-color_($color) {
   .mdc-bottom-line { 
      @include mdc-bottom-line-color($color);
    }
}
``` 
Otherwise the user will require a separate class to color this component, which is essentially still a subcomponent.